### ISSUE TITLE
Pie Chart Fill Color Looks Different with Different Android Versions

### ISSUE THREAD

**Issue by robinsun1986** [PhilJay/MPAndroidChart#2555]
I've got a weird issue where the same code produces pie charts with totally different fill colors (seemingly) on different Android versions. Specifically, (<=4.3) and (>=5.0) looks different for **#F6F8F8** and **#E4F1FF**.  I was initially using MPAndroidChart v2.2.5 and then upgraded to v3.0.1. Both got the same issue.

It's not a device specific issue as I tested on simulators (4.3 and 6.0) on the same mac. 
Neither is it a simulator issue as I can reproduce this on real devices (4.2.2 and 5.0.1). 
The color difference is big.

Basically, it looks like this on v5.0+ devices.
![screen shot 2016-12-07 at 4 13 02 pm](https://cloud.githubusercontent.com/assets/8778657/20955542/1d9d4a2c-bc98-11e6-8a89-2641d549f0bb.png)

On v4.3 or below devices, it looks like this.
![screen shot 2016-12-07 at 4 08 52 pm](https://cloud.githubusercontent.com/assets/8778657/20955552/2ce9782a-bc98-11e6-842a-3c116bb5b15b.png)
